### DISCUSSION TITLE
How to write unit tests against proxy contract with `taq test` command?

### DISCUSSION THREAD

**Discussion by mastercodercat** [ecadlabs/taqueria#1905] [Q&A]
I wrote the NFT collection smart contract with jsligo, and converted it to using proxy contract for upgradability using this tutorial from marigold dev.
However, I faced difficulties while writing unit tests against proxy contract using taqueria. Can anyone help me for this issue?

**Comment by mweichert** [comment]
HI @mastercodercat do you have a repo we can check out?
